src/main/java/com/opencart/steps/AddToShoppingCartBL.java

```diff
@@ -0,0 +1,22 @@
+package com.opencart.steps;
+
+import com.opencart.pages.AddToShoppingCartPage;
+import org.openqa.selenium.WebElement;
+
+public class AddToShoppingCartBL {
+    private AddToShoppingCartPage addToShoppingCartPage;
+
+    public AddToShoppingCartBL() {
+        addToShoppingCartPage = new AddToShoppingCartPage();
+    }
+
+    public AddToShoppingCartBL clickOnAddToShoppingCartButton() {
+        addToShoppingCartPage.getAddToShoppingCartButton().get(0).click();
+        return this;
+    }
+
+    public ShoppingCartBL clickOnShoppingCartButton() {
+        addToShoppingCartPage.getShoppingCartButton().click();
+        return new ShoppingCartBL();
+    }
+}
```

Comment:
I think you can move it HeaderPageBL business logic because you can get access to the shopping cart button from any page